src/Tree.tsx

```diff
@@ -32,6 +32,7 @@ type Props = {
   filesChanged: string[];
   maxDepth: number;
   colorEncoding: "type" | "number-of-changes" | "last-change"
+  customizeFileColors?: { [key: string]: string };
```

Comment:
let's mirror the change above - this should be `fileColors`

Reply:
but I find,
```
const fileColors = { ...defaultFileColors, ...customizeFileColors };
```
this variable has existed, so I create a new one, so Is enough if we only change the user input (customize_file_colors => file_colors), and I guess the user does not care what name in our code

Reply:
> let's mirror the change above - this should be `fileColors`

but I find,
```
const fileColors = { ...defaultFileColors, ...customizeFileColors };
```
this variable has existed, so I create a new one, Is it enough if we only change the user input (customize_file_colors => file_colors)? and I guess the user does not care what name in our code

Reply:
yep, makes sense! I like how you've changed things - one last change that might be helpful: do you mind changing `customizeFileColors` to `customFileColors` to be a bit more clear?

Reply:
No problem, done